apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -280,6 +297,16 @@ struct ComposeView: View {
                 }
             }
         }
+        .onChange(of: composeState.contextItem) { _ in
+            if composeState.liveMessage?.sent == false {
+                chatModel.removeLiveChatItemDummy()
+                var quoted: ChatItem? = nil
+                if case let .quotedItem(item) = composeState.contextItem {
+                    quoted = item
+                }
+                let _ = chatModel.addLiveChatItemDummy(quoted, chat.chatInfo)
+            }
+        }
```

Comment:
this should not be a change listener, as this can only happen during a function call that you control - it should happen in that function